middleware.js

```diff
@@ -42,7 +42,9 @@ export function middleware (request) {
     // unsafe-inline for styles is not ideal but okay if script-src is using nonces
     "style-src 'self' a.stacker.news 'unsafe-inline'",
     "manifest-src 'self'",
-    'frame-src www.youtube.com platform.twitter.com rumble.com',
+    // Using wildcard (*) is the only way to get peertube videos working
+    // as the domain for each is different
+    'frame-src *',
```

Comment:
If this is deemed too much of a security risk then we will probably need to cancel this PR (unless there's a way to dynamically create the frame-src value. I did try `peertube.*` and `peertube.*be` etc but these don't seem to be valid.

Reply:
Makes sense. [I just learned that one can sandbox iframes](https://web.dev/articles/sandboxed-iframes). For PeerTube embeds, only `allow-scripts` seems to be required (see suggestion). With these restrictions, I think it's okay to use a wildcard on `frame-src`.

I also think `frame-src` isn't actually that important since we don't allow arbitrary iframes anyway, we only use iframes for the origins that our code parses which we therefore already inherently trust. So it might have been just a nice layer of defense. Additionally, according to my research, iframes are bad if _other_ sites frame your content in a malicious way which we don't allow via the `X-Frame-Options: DENY` header.

Reply:
Sounds like a "go" then 😄